content/docs/app-developer-guide/build-a-windows-app.md

```diff
@@ -0,0 +1,87 @@
++++
+title="Build a Windows app"
+weight=2
+summary="The basics of taking your Windows app from source code to runnable image."
+aliases=[
+    "/docs/using-pack/building-windows-app/"
+]
```

Comment:
```suggestion
```

Since this is a new page that never existed under `using-pack`, this shouldn't be necessary.

Reply:
Ah I see. Will remove!